src/components/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Formik, Form } from "formik";
+import * as Yup from "yup";
+import { toast } from "react-toastify";
+import { useTranslation } from "react-i18next";
+import { Link } from "react-router-dom";
+
+import { Label, IconText, LogoPage, Toast, Line, InputAtCard } from "components/styles";
+import { ButtonForm, ForgotPasswordForm, StyledInlineErrorMessageForm, Footer, CardForm } from "./ForgotPassword.style";
+import { sendEmail } from "services/user.service";
+import { paths } from "config/paths";
+import IForgotPassword from "./IForgotPassword.interface";
+
+export const ForgotPassword = () => {
+  const { t } = useTranslation();
+
+  const schema = Yup.object().shape({
+    email: Yup.string()
+      .required(t`user.email.validation.requied`)
+      .email(t`user.email.validation.format`),
+  });
+
+  const initialValues: IForgotPassword = {
+    email: ""
+  };
+
+  return (
+    <>
+      <Formik
+        validationSchema={schema}
+        initialValues={initialValues}
+        onSubmit={(values) => {
+          sendEmail(values).then(
+            (res) => {
+              toast.success(t`toast.sendEmail.success`);
+            },
+            (error) => {
+              const resMessage = (error.response && error.response.data) || error.message || error.toString();
+              switch (resMessage) {
+                case "User not found":
+                  return toast.error(t`toast.sendEmail.notFound`);
+                default:
+                  return toast.error(t`toast.sendEmail.error`);
+              }
```

Comment:
jako case nie używaj stringa, api powinno zwrócić kod np. 200 jeśli wszystko jest OK, lub 400 jak coś jest nie tak

Reply:
Niestety to są rzeczy z serwera i żeby były przetłumaczone to muszę użyć swicha bo są różne komunikaty błędów

Reply:
![obraz](https://user-images.githubusercontent.com/56917808/162922632-d6248e30-f6eb-4c1a-97be-c4f3ccb45c28.png)

Reply:
user not found zwraca kod 404
inny błąd zwraca 400